scripts/reduce_CG1D.py

```diff
@@ -11,16 +11,14 @@
 
 # standard imports
 from datetime import datetime
-import logging
 from pathlib import Path
 from typing import Union
 import sys
 
 # declare the conda environment for this to run in
 CONDA_ENV = "imars3d-dev"
-ERROR_GENERAL = 1  # if more errors, they could be turned into enum.Enum
+ERROR_GENERAL = -1  # for things that aren't workflow
```

Comment:
I thought exit codes had to be greater or equal than zero

Reply:
linux (and everything else) turns -1 the highest int used for exit codes